MoneySaver.Api/Controllers/CategoryController.cs

```diff
@@ -38,27 +38,27 @@ IEnumerable<TransactionCategoryModel> result
         }
 
         [HttpGet("{transactionCategoryId}")]
-        public IActionResult GetCategory(int transactionCategoryId)
+        public async Task<IActionResult> GetCategoryAsync(int transactionCategoryId)
         {
-            return this.Ok(this.categoryService.GetCategory(transactionCategoryId));
+            return this.Ok(await this.categoryService.GetCategoryAsync(transactionCategoryId));
```

Comment:
You can put "await this.categoryService.GetCategoryAsync(transactionCategoryId)" on a separate row in order to be easier for debug and more readable.
for example:
var result = await this.categoryService.GetCategoryAsync(transactionCategoryId)
return this.Ok(result)